filebeat/input/filestream/internal/input-logfile/manager.go

```diff
@@ -179,10 +179,12 @@ func (cim *InputManager) Create(config *conf.C) (v2.Input, error) {
 
 	cim.idsMux.Lock()
 	if _, exists := cim.ids[settings.ID]; exists {
-		cim.Logger.Errorf("filestream input with ID '%s' already exists, this "+
+		cim.Logger.Warnf("filestream input with ID '%s' already exists, this "+
```

Comment:
This is still an error. It is a common user error when writing a standalone Filebeat configuration and it does lead to data duplication.

Unfortunately we cannot make an unique ID required as it is a breaking change, so the log error is the best way to tell our users their configuration is going to cause trouble.